htdocs/core/actions_massactions.inc.php

```diff
@@ -49,6 +49,18 @@
 @phan-var-force ?string $deliveryreceipt
 ';
 
+'
+@phan-var-force ?string $permissiontoread
```

Comment:
Seems duplicate of line 41 to 49

Reply:
Possibly because this PR and changes still pending here are "old" and repeated in more recent PRs.  I'll clean that up.

The main reason this PR is still open is for these phan notices:
```plaintext
Warning: actions_massactions.inc.php: PhanUndeclaredProperty: Reference to undeclared property \CommonObject-&gt;fk_soc
Warning: actions_massactions.inc.php: PhanUndeclaredProperty: Reference to undeclared property \User-&gt;email_aliases
Warning: actions_massactions.inc.php: PhanPossiblyUndeclaredGlobalVariable: Global variable $pagecount is possibly undeclared
```

I am tending to think that I should simply extract the code changes to a separate PR and close this one - the real technical debt is likely not going to be fixed until almost all of the pending phpdoc changes and false positives are identified.